bitnami/harbor/values.yaml

```diff
@@ -311,6 +314,8 @@ service:
       # The service port Notary listens on. Only needed when notary.enabled
       # is set to true
       notaryPort: 4443
+    annotations: {}
```

Comment:
The `loadBalancer` section in the _values-production.yaml_ doesn't contains those values

Reply:
Good point, it seems I added them to the wrong location.